simplecloud-plugin/build.gradle

```diff
@@ -46,7 +46,6 @@ shadowJar {
     archiveFileName.set("SimpleCloud-Plugin-${version}.jar")
     dependencies {
         relocate 'com.google.gson', 'eu.thesimplecloud.external'
-        relocate 'kotlin', 'eu.thesimplecloud.external'
```

Comment:
What does this do here?

Reply:
because I accidentally used an old branch